frontend/src/hooks/User/users.js

```diff
@@ -0,0 +1,26 @@
+import useSWR from "swr";
+
+import axios from "../../lib/axios";
+
+export const useUserList = () => {
+  const { data: users, error } = useSWR(
+    "/api/user/user",
+    () =>
+      axios
+        .get("/api/user/user")
```

Comment:
Pls make a way to update just using below.
```suggestion
        .get("/api/user")
```

Since you already have prefix for `/user` in your routes, try like this?

```
...
Route::apiResource('/', UserController::class)
...
```